parcels/tools/converters.py

```diff
@@ -53,9 +53,19 @@ def reltime(self, time):
             return (time - self.time_origin) / np.timedelta64(1, 's')
         elif self.calendar in _get_cftime_calendars():
             if isinstance(time, (list, np.ndarray)):
-                return np.array([(t - self.time_origin).total_seconds() for t in time])
+                try:
+                    return np.array([(t - self.time_origin).total_seconds() for t in time])
+                except np.core._exceptions.UFuncTypeError:
+                    raise ValueError(f"Cannot subtract 'time' (a {type(time)} object) from "
+                                     f"a {self.calendar} calendar.\n"
+                                     f"Provide 'time' as a {type(self.time_origin)} object?")
```

Comment:
this string-concatenate generates a syntax-error - isn't there a '\' missing at the end of the line ?

Reply:
Not sure why this breaks PEP8. The code works in my local system, and `flake8` also doesn't throw an error then. I could try going back to old-skool `%` formatting?

Reply:
I've now used %-strings again